modules/admin/admin-ui/src/main/resources/web/admin/common/js/liveedit/ContentBackedComponentView.ts

```diff
@@ -0,0 +1,59 @@
+module api.liveedit {
+
+    import Component = api.content.page.region.Component;
+    import ContentSummary = api.content.ContentSummary;
+    import ContentSummaryBuilder = api.content.ContentSummaryBuilder;
+    import ContentSummaryAndCompareStatus = api.content.ContentSummaryAndCompareStatus;
+    import ContentTypeName = api.schema.content.ContentTypeName;
+
+    export class ContentBackedComponentViewBuilder<COMPONENT extends Component> extends ComponentViewBuilder<COMPONENT> {
+
+        contentTypeName: ContentTypeName;
+
+        constructor() {
+            super();
+        }
+
```

Comment:
It's not necessary to have an empty constructor with only call to ``super()``. Maybe we should pass ``contentTypeName`` as an argument to the constructor since it's a mandatory property for the class.